qutebrowser/browser/commands.py

```diff
@@ -853,7 +853,7 @@ def zoom_out(self, count=1):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('count', count=True)
-    def zoom(self, zoom: int=None, count=None):
+    def zoom(self, zoom: int = None, count=None):
```

Comment:
Pylint's default settings want spaces between annotated key arguments, are you :+1: on that?